README.md

```diff
@@ -126,13 +126,13 @@ You can set a timeout per message
 import { NotificationDisplay, notifier } from '@beyonk/svelte-notifications'
 
 function someFunction () {
-  notifier.success('Notifications work!', 5000) // built in theme
-  notifier.send('custom-theme', 'Notifications work!', 5000) // custom theme
+  notifier.success('Notifications work!', { duration: 5000 }) // built in theme
```

Comment:
Unfortunately this significantly changes the core API. I like the idea of an options object in principle, but this decision should be made in a completely separate PR which only modifies the API, not muddled in among some refactorings and a new feature.